pkgs/development/tools/symfony-cli/default.nix

```
@@ -0,0 +1,29 @@
+{ stdenvNoCC, fetchurl, lib, testVersion, symfony-cli, unzip }:
```

```suggestion
{ stdenvNoCC, fetchurl, lib }:
```

remove the unused arguments.